### PR TITLE
[aria2] Add Chart

### LINKED ISSUE #1210: Add Aira2c Chart
# Details

**Helm chart name: Aria2c**

**Describe the solution you'd like:** New Chart

**Anything else you would like to add:** Ref https://hub.docker.com/r/p3terx/aria2-pro

**Additional Information:** Self assign to @rwaltr

### PR DESCRIPTION
**Pull request by rwaltr**
- Added base aria2 chart
- Added README

**Description of the change**

Added new Chart

**Benefits**

New Possibilities

**Possible drawbacks**

Moar dependencies

**Applicable issues**

- fixes #1210

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by rwaltr** [comment]
Ready for review!

File: charts/stable/aria2/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+appVersion: 1.0.0
+description: aria server for downloading web content
+name: aria2
+version: 2.0.1
```
Comment (onedr0p) [inline charts/stable/aria2/Chart.yaml:5]: This should be 1.0.0

File: charts/stable/aria2/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+appVersion: 1.0.0
```
Comment (onedr0p) [inline charts/stable/aria2/Chart.yaml:2]: This should be the application version

**Review by onedr0p — CHANGES_REQUESTED** [review]
Minor changes

File: charts/stable/aria2/Chart.yaml (hunk 2 above)
Comment (rwaltr) [reply charts/stable/aria2/Chart.yaml]: upsteam doesnt use semver, attempted "Semvered" date based version

**Review by bjw-s — APPROVED** [review]

**Pull request merged**